src/plone/restapi/tests/test_dxcontent_serializer.py

```diff
@@ -498,6 +503,18 @@ def test_allow_discussion_obj_instance_allows_global_enabled(self):
         self.assertIn("allow_discussion", obj)
         self.assertEqual(True, obj["allow_discussion"])
 
+    @unittest.skipUnless(HAS_PLONE_6, "Since Plone 6 the Plone site is indexed ...")
+    def test_allow_discussion_portal_default(self):
+        """Not globally addable, not fti enabled, not obj instance enabled"""
+        if HAS_PLONE_6:
```

Comment:
This test is not necessary. The decorator already skips the test if the condition is not met.